pkg/kubelet/cm/dra/manager.go

```diff
@@ -268,17 +268,17 @@ func (m *Manager) prepareResources(ctx context.Context, pod *v1.Pod) error {
 			return fmt.Errorf("ResourceClaim %s: %w", resourceClaim.Name, err)
 		}
 		infos[i].claimInfo = claimInfo
-		infos[i].drivers = make(map[string]*draplugin.Plugin, len(claimInfo.DriverState))
+		infos[i].plugins = make(map[string]*draplugin.Plugin, len(claimInfo.DriverState))
 		for driverName := range claimInfo.DriverState {
-			if plugin := infos[i].drivers[driverName]; plugin != nil {
+			if plugin := infos[i].plugins[driverName]; plugin != nil {
 				continue
 			}
-			client, err := m.draPlugins.NewDRAPluginClient(driverName)
+			plugin, err := m.draPlugins.GetDRAPlugin(driverName)
```

Comment:
Is there any documentation that explains that DRA plugin is a part of Kubelet and DRA driver is an external component to which DRA plugin is connecting?

Reply:
The terminology used elsewhere is that a DRA driver "provides (or deploys) a kubelet plugin".

I don't think we say that "DRA plugin is a part of Kubelet" or "DRA driver is an external component to which DRA plugin is connecting".

Reply:
I am fine with / actually prefer the term plugin here. The driver is the whole thing -- this is the kubelet plugin part of that.